library/ssl_tls13_client.c

```diff
@@ -666,6 +666,75 @@ int mbedtls_ssl_tls13_write_client_hello_exts( mbedtls_ssl_context *ssl,
 /*
  * Functions for parsing and processing Server Hello
  */
+static int ssl_tls13_is_supported_versions_ext_present(
```

Comment:
Please add some documentation, at least about the return value (I think it's 1 if present, 0 if absent, negative if an error occurred).